src/style.css

```diff
@@ -0,0 +1,117 @@
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: stretch;
+}
+
+.form-big {
+  padding-top: 20px;
+  margin-bottom: 20px;
+  background: #fff;
+}
+
+#bigSearch {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 25px;
+}
+
+.keypress {
+  width: 100%;
+  height: 70px;
+  left: 48px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  padding: 4px 9px 4px 62px;
+}
+
+.keypress::placeholder {
+  position: absolute;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  left: 62px;
+  top: 35.71%;
+}
+
+.keypress:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.keypress:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.form-small {
+  margin-bottom: 20px;
+  background: #fff;
+  position: relative;
+}
+
+#smallSearch {
+  background-image: url("images/Search.svg");
```

Comment:
Don't repeat similar styles. Follow the checklist, please 
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/99403198/234832627-00ab5e34-a2b1-4d44-b913-4bdc2a9d2760.png">